spec/system/user_creates_an_article_spec.rb

```diff
@@ -10,6 +10,7 @@
 
     expect(page).to have_content "Article was successfully created"
     article = Article.last
+    expect(article.user_id).to eq(user.id)
```

Comment:
**suggestion (blocking):** Write a different test.

Instead of focusing on some internal state, it may be better to focus on something a user can see. A way to achieve this is to write specs that test when a user tries to do something inaccessible to them. What happens when they try to edit or view someone else's article?